### DISCUSSION TITLE
How one can import DUMMY_HTTP_TS?

### DISCUSSION THREAD

**Discussion by dimkar121** [MatthiasValvekens/pyHanko#246] [Q&A]
Hallo,
How one can import ```DUMMY_HTTP_TS``` in order to save some time during signing in the interrupted version?

my best regards,
Dimitrios

**Comment by MatthiasValvekens** [comment]
`DUMMY_HTTP_TS` is a constant/fixture in the tests, not part of the library. What are you trying to achieve? If you don't particularly care about timestamps and it's slowing you down, you can always opt _not_ to pass in a `Timestamper`... ;)

**Reply by dimkar121** [reply]
I am using the interrupted signing and it looks like it invokes the timestamping process twice (```prepare_document()``` and ```finish_signing()```). 
Which part is necessary? If I called it only in ```prepare_document()```, then it does not actually sign the document.
